src/translations/screens/subscreens/Login.ts

```diff
@@ -0,0 +1,54 @@
+import {translation as _} from '../../commons';
+const LoginTexts = {
```

Comment:
@magnusolderoesaether @pettersuul 
You can take a look at the texts and the translations in this file.